Dockerfile

```diff
@@ -2,11 +2,9 @@ FROM alpine:3.4
 
 MAINTAINER Geshan Manandhar (http://geshan.com.np)
 
-RUN apk --update add curl &&\
-    apk add fabric --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted &&\
-    rm /var/cache/apk/*
+RUN apk add --no-cache curl rsync openssh &&\
+    apk add --no-cache fabric --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted
```

Comment:
> please add back rm /var/cache/apk/* 

Thanks to `--no-cache` option, no need to `rm /var/cache/apk/*`.
You can check it like this.
`docker run -it cc01a56f105d ls /var/cache/apk`

> rebase with latest master.

Understood. I will ping you when it's finished.

Thank you.